### ISSUE TITLE
Version 8 api breaks wake_up command

### ISSUE THREAD

**Issue by rs333** [timdorr/tesla-api#203]
After the api_version 8 update from 6/29/2020, the wake_up api command requires a ? after the word wake_up to work for 2018 Model AWD 3s.  It may also be required for other vehicles.  This is not indicated in the documentation and was not previously required.

 `/api/1/vehicles/{id}/wake_up?`

This suggests there may be other parameters.

Note that until you update the Tesla app on your phone, the Tesla app may also have difficulty waking your car.

**Comment by timdorr** [comment]
I'm not sure why that would matter, since it wouldn't affect URL parsing and result in a different request action. No parameters are indicated by the app code. I think it's just a behavior thing, based on when and how often you call it.

**Comment by rs333** [comment]
> 
> 
> I'm not sure why that would matter, since it wouldn't affect URL parsing and result in a different request action. No parameters are indicated by the app code. I think it's just a behavior thing, based on when and how often you call it.

I agree it doesn't make sense.  I can say that on two separate Model 3's, they would not wake from being in asleep without the question mark or without the latest version of the Telsa App.  Curious if it works on your vehicles.

**Comment by timdorr** [comment]
The Tesla app being the latest version or not has no difference in how your direct use of the API behaves. The two are separate things. The app doesn't affect the vehicle state in any way. 

Is this after installing the 2020.24 update? I haven't gotten it on my vehicle yet, so I don't have any way of confirming a change. But wake_up continues to work just fine for my P90D.

**Comment by rs333** [comment]
The app says both our cars are Version 2020.20.12 d2c8a3e110f4.  The App only makes a difference in that, after updating the Tesla App, the Tesla app started waking waking the car without issue.  Prior to updating it, the two cars refused to wake. 

If you look at the response to the wake_up command, it reports `api_version: 8`.  Prior to yesterday, and within your examples, I believe it was `api_version: 7`.  It almost looks like they updated the API without doing a full vehicle update.  Unexpected, but theoretically possible.  My app was working for me up until 7:00AM EST on the 29th.  I noticed issues later in the day and after much trouble shooting, that included using the official Tesla App to try and wake the cars, I decided to check to see if there was an update in my phone.  There was.  As soon as I updated the Tesla app on the phone (Android for one and iOS for the other), the Tesla app woke the cars without any issues.  My personal app was giving the following errors to the wake_up command. ` api/1/vehicles/123412431243/wake_up`  (Id changed for security reasons)

1st try
` error: "vehicle unavailable: {:error=>\"vehicle unavailable:\"}"`
2nd try shortly after 1st try. 
` error: "not_found"`

The get_vehicle command worked fine.  Presumably because it doesn't talk to the car.  On a whim, I added the question mark to the `wake_up` command thinking maybe they tweaked something.  It could be with the intent of adding parameters to the `wake_up` command such as how long to wake for or it could just be a bug in their api.  After adding the `?` all worked perfectly.  One key difference compared to the last time I looked at the raw json responses, Tesla now reports api_version=8 as opposed to 7.

It is curious.  Does your report api_version 8 or 7?

P.S.  I appreciate your responses and the work you have done documenting the api.  It has been very useful.
